packages/pods-core/src/basev3.ts

```diff
@@ -225,37 +229,19 @@ export abstract class ExportPod<
 > {
   public L: DLogger;
   static kind = "export" as PodKind;
-  get config(): PodConfig[] {
-    return [
-      {
-        key: "dest",
-        description: "Where to export to",
-        type: "string" as const,
-        required: true,
-      },
-      {
-        key: "includeBody",
-        description: "should body be included",
-        default: true,
-        type: "boolean",
-      },
-      {
-        key: "includeStubs",
-        description: "should stubs be included",
-        type: "boolean",
-      },
-    ];
-  }
+  abstract get config(): JSONSchemaType<T>;
+
   constructor() {
     this.L = createLogger("ExportPod");
   }
 
   validate(config: Partial<T>) {
-    const { dest } = config;
-    const configJSON = JSON.stringify(config);
-    if (_.isUndefined(dest)) {
+    const validateConfig = ajv.compile(this.config);
+    validateConfig(config);
+    if (!validateConfig(config)) {
       throw new DendronError({
-        message: `no dest specified. config: ${configJSON}`,
+        message: "validation errors",
+        payload: "error",
```

Comment:
can we return the list of errors here as well in `payload`?

Reply:
Yes. Adding in message as well, since only message is visible to user.